src/views/DefineCompany.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div>
     <div class="mt-10 company-statement">
-      <p v-if="getCompanyResources.title">
-        <span class="company-statement-label">{{ getCompanyResources.title }}</span>
-        {{ getCompanyResources.description }}
+      <p v-if="getCompanyTitle">
+        <span class="company-statement-label">{{ getCompanyTitle }}:</span>
```

Comment:
I changed this (added v-if above and colon at the end of this line) to match the same code in ReviewConfirm.vue.

Reply:
Oops, I meant that I changed ReviewConfirm.vue to match THIS file.

Reply:
Cam, is this OK?